javascript/features/death_match/death_match_stats.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+export class DeathMatchStats {
+    constructor() {
+        this.kills_ = 0;
+        this.deaths_ = 0;
+        this.damage_ = 0;
+
+        this.bulletsHit_ = 0;
+        this.bulletsMissed_ = 0;
+    }
+
+    get kills() { return this.kills_; }
+    get deaths() { return this.deaths_; }
+    get kdRatio() { return this.deaths === 0 ? this.kills : Math.round(this.kills / this.deaths * 100) / 100; }
+    get damage() { return this.damage_; }
+    get bulletsHit() { return this.bulletsHit_; }
+    get bulletsMissed() { return this.bulletsMissed_; }
+    get accuracy() {
```

Comment:
Might be worth suffixing this with `Percentage`, as it returns a different unit from `kbRatio`